homework.md

```diff
@@ -86,28 +87,28 @@ You send the changes of your local repository to Github
 
 What is Github?
 
-A place for us to store and share our code.
+is a source control platform use when created code .is a store place where you can save your project
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+is to solve the error your change to the repository or stash your change later
 
```

Comment:
to put it simply, "to commit your changes" is a Github way of saying to "save your changes". And, don't forget that it is good practice to always give a name to your changes and a brief description before you commit.